Documentation/spec/docfx_document_schema.md

```diff
@@ -204,4 +247,13 @@ Here's the schema to describe these operations:
         }
     }
 }
-```
\ No newline at end of file
+```
+
+## 8. Open Issues
+1. DocFX fills `_global` metadata into the processed data model, should the schema reflect this behavior?
```

Comment:
`_global` metadata is one of the outputs of processor, correct? If so, a separate doc looks better to me. Or, shall we consider also have a schema for outputs?